packages/clarity-js/src/layout/mutation.ts

```diff
@@ -124,12 +124,13 @@ async function process(): Promise<void> {
   task.start(timer);
   while (mutations.length > 0) {
     let record = mutations.shift();
+    let instance = time();
```

Comment:
>instance [](http://example.com/codeflow?start=8&length=8)

Minor: Name is odd-seeming to me as instance to me implies some sort of object, not a time. Did you mean instant?

Reply:
It may not be the best choice for the word but I did mean instance -- as in single occurrence of something. The idea is that we don't want to suspend multiple mutations which occur together since that wouldn't really generate any additional chatter over the wire.